packages/fast-components-react-base/app/index.tsx

```diff
@@ -64,6 +64,7 @@ function render(): void {
             formPlugins={formPlugins}
             showTransparencyToggle={true}
             styleEditing={true}
```

Comment:
Whats the difference between `styleEditing` and `stylePropertyEditing`?

Reply:
It's the ability to edit styles from the CSS editor, and the ability to edit the individual style properties. I *could* just have the one and you get both, it's very "internal use only" right now.

Reply:
You know what, because this is internal use, I'll forgo adding another property since both of these will always be true.